src/components/__tests__/ContextView.ts

```diff
@@ -0,0 +1,29 @@
+import { cursorDown, newThought } from '../../action-creators'
+import { store } from '../../store'
+import createTestApp, { cleanupTestApp } from '../../test-helpers/createRtlTestApp'
+import { findThoughtsByText } from '../../test-helpers/queries'
+import { setCursorFirstMatchActionCreator } from '../../test-helpers/setCursorFirstMatch'
+
+describe('Extract thought', () => {
+  beforeEach(async () => {
+    await createTestApp()
+  })
```

Comment:
This can be simplified to `beforeEach(createTestApp)` since the return value is not being used.